src/scripts/standalone.js

```diff
@@ -0,0 +1,3 @@
+window.appMode = 'standalone';
+
+import('./site');
```

Comment:
I believe this whole file can be deleted now.

Reply:
this file is used if you want to run the command yarn serve, so you can use it locally for development purposes.